### PR TITLE
Add checking for extension methods

### PR DESCRIPTION
**Pull request by YairHalberstadt**

### CODE REVIEW

File: ConfigureAwaitChecker.Lib/Checker.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public static bool CanHaveConfigureAwait(ExpressionSyntax expression, SemanticMo
 			var type = typeInfo.ConvertedType;
 			if (type == null)
 				return false;
-			var members = type.GetMembers(ConfigureAwaitIdentifier);
+			var members = semanticModel.LookupSymbols(expression.SpanStart, type, "ConfigureAwait", true);
```
Comment (cincuranet) [inline ConfigureAwaitChecker.Lib/Checker.cs]: The `ConfigureAwaitIdentifier` should be used here.

Comment (YairHalberstadt) [reply ConfigureAwaitChecker.Lib/Checker.cs]: Fixed

**Comment by YairHalberstadt** [comment]
Do you think you'd be able to have another look at this at some point @cincuranet?
Thanks :-)

**Comment by cincuranet** [comment]
Yeah, hopefully next week.

**Comment by YairHalberstadt** [comment]
Thanks

**Pull request merged**